src/Collections/Map.php

```diff
@@ -266,4 +266,17 @@ public function get($key, $default)
         [$exists, $value] = static::apply($this->data, $key, 0, null);
         return $exists ? $value : $default;
     }
+
+    /**
+     * Returns a copy of Map as PHP associative array.
+     *
+     * @return array<mixed, mixed>
+     */
+    public function toArray()
```

Comment:
Do we want to be extra explicit and call this `toAssociateArray`?

Reply:
I'd support that, yeah.

Reply:
Same thought. I named it `toArray` so that it's similar to Vector API. But since you folks also think this should be `toAssociativeArray`, then why not! Updated.

Reply:
yes that was a typo, I meant `toAssociativeArray` (or `toAssocArray`). Thank you!